Formula/par2.rb

```diff
@@ -1,18 +1,19 @@
 class Par2 < Formula
   desc "Parchive: Parity Archive Volume Set for data recovery"
   homepage "https://github.com/Parchive/par2cmdline"
-  url "https://github.com/Parchive/par2cmdline/releases/download/v0.7.4/par2cmdline-0.7.4.tar.bz2"
-  sha256 "e602db3d8bdc49e2cb9e0e089ec31cd262e661ef7450f5d556e43a97a299e71d"
+  url "https://github.com/Parchive/par2cmdline/releases/download/v0.8.0/par2cmdline-0.8.0.tar.bz2"
+  sha256 "496430e185f2d82e54245a0554341a1826f06c5e673fa12a10f176c7f9b42964"
 
-  bottle do
-    cellar :any_skip_relocation
-    sha256 "a825515ff251975d362998560f5e2e046ce3a9f753be106bbf3717c6d411b7fb" => :high_sierra
-    sha256 "6c3432e8a1b7e8ceeabb380af04d13123c9fb542fab7caf62fe8201f3d1adee2" => :sierra
-    sha256 "7257b39640dcf1894c2329129406a573b04dd263bf3b06283dc854b5ed17cf8e" => :el_capitan
-  end
+  option "with-llvm", "(Use LLVM to compile and enable OpenMP support)"
+  depends_on "llvm" => [:build, :optional]
```

Comment:
We shouldn't depend on `llvm`, but use: `option "with-openmp", "Compile with OpenMP support"` and `needs :openmp if build.with? "openmp"` which handles it automatically. See for example how `imagemagick` does it.